test/TensorFlow/sr8426.swift

```diff
@@ -0,0 +1,49 @@
+// RUN: %target-swift-frontend -Xllvm -tf-dump-intermediates -Onone -emit-sil -verify %s
```

Comment:
we have a new test case called optimization-disabled. Should we add this test there?

Reply:
good idea. Moved the test there and removed this file.